topydo/lib/Command.py

```diff
@@ -92,3 +96,64 @@ def usage(self):
     def help(self):
         """ Returns the help text for this command. """
         raise NotImplementedError
+
+
+class WriteCommand(Command):
```

Comment:
Please move this class to a separate file (along with the introduced imports above).